src/components/views/JobInfo.js

```diff
@@ -78,11 +77,11 @@ export default class JobInfo extends React.Component {
 
   getActionMenu = ({ onToggle }) => {
     const { data: { job } } = this.props;
-    const isJobNotInProgress = get(job, 'status.value') !== 'in_progress';
+    const isJobNotInProgress = job?.status?.value !== 'in_progress';
 
     return (
       <IfPermission perm="ui-local-kb-admin.jobs.delete">
-        <React.Fragment>
+        <>
```

Comment:
Looks like we don't even need a Fragment here since there's only a single component in it.